boards/reel_board.json

```diff
@@ -0,0 +1,44 @@
+{
+  "build": {
+    "core": "nRF5",
+    "cpu": "cortex-m4",
+    "f_cpu": "64000000L",
+    "mcu": "nrf52832",
+ 	"hwids": [
+		[
+        "0x0D28",
+        "0x0204"
+      	]
+    ],
+   "variant": "reel_board"
```

Comment:
Please move the `variant` field to the `zephyr` section as it done in [this](https://github.com/platformio/platform-nordicnrf52/blob/develop/boards/laird_bl652_dvk.json#L7) manifest.